seenanadimi/react-challenge-project-sep-trial
# bonus(login #11): added register functionality and login check

## Changes
_List Changes Introduced by this PR_
1. Added register functionality and login check

## Testing Steps
_How do the users test this change?_
Register new account / login with new account. Check localhost:4000/api/user-list to check for new registered account.

## Learning
_Describe the research stage_
Researching useful examples of successful logins.

_Links to blog posts, patterns, libraries or add-ons used to solve this problem_

_If this closes an issue, name it here. If it doesn't, remove this line_
Closes #13
